CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## [2.0.0] (Unreleased)
 
+* **Fix**: [177](https://github.com/SimformSolutionsPvtLtd/flutter_chatview/pull/177) Fixed
```

Comment:
This would be a breaking change as you have changed how a model would parse a json and output to a json.